openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +484,53 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is handled by the Request URI endpoint of the Verifier.  
+
+The request MUST use the HTTP POST method with the https scheme the content type "application/x-www-form-urlencoded" and the accept header set to "application/oauth-authz-req+jwt".
+
+The following parameters are defined: 
+
+`wallet_metadata`:
+: OPTIONAL. A String containing a value the Verifier MUST use in the `wallet_nonce` authorization parameter when creating the signed presentation request object. For example, a base64url encoded fresh, cryptographically random number with sufficient entropy. 
```

Comment:
Wait... the parameter is named wallet_metadata but the text reads as though it's about the nonce.  And wallet_nonce is the next param.

Reply:
wow, yes, it seems quite obvious.

Reply:
Looks like this was an accidental mix-up in one of the latest commits: https://github.com/openid/OpenID4VP/pull/59/commits/e1276ec43e7b7de4fa5849a5f79fd79c5267ae28

Reply:
fixed